chart/compass/templates/system-fetcher-configmap.yaml

```diff
@@ -0,0 +1,190 @@
+{{if .Values.global.systemFetcher.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: compass-system-fetcher-templates-config
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: {{ .Chart.Name }}
+    release: {{ .Release.Name }}
+data:
+  integration-systems.json: |-
```

Comment:
This goes for the Int systems and app templates - can we only define the keys 'integration-systems.json' and 'templates.json' as they are here, but have their values (the big chunks of raw json) as placeholder configuration values?

The point is that the current values won't make sense for the open source. They are SAP-specific, hence it would be best to have them defined only in our internal mgt-plane-config repo as overrides.